### ISSUE TITLE
关键信息抽取re训练加载数据之后训练自行结束，没有报错信息。

### ISSUE THREAD

**Issue by smallhaozigithub** [PaddlePaddle/PaddleOCR#9732]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：aistudio
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：!python tools/train.py -c configs/kie/vi_layoutxlm/ser_vi_layoutxlm_lxsxkz_zh.yml
- 完整报错/Complete Error Message：
- [2023/04/14 16:25:17] ppocr INFO: Architecture : 
[2023/04/14 16:25:17] ppocr INFO:     Backbone : 
[2023/04/14 16:25:17] ppocr INFO:         checkpoints : None
[2023/04/14 16:25:17] ppocr INFO:         mode : vi
[2023/04/14 16:25:17] ppocr INFO:         name : LayoutXLMForRe
[2023/04/14 16:25:17] ppocr INFO:         pretrained : True
[2023/04/14 16:25:17] ppocr INFO:     Transform : None
[2023/04/14 16:25:17] ppocr INFO:     algorithm : LayoutXLM
[2023/04/14 16:25:17] ppocr INFO:     model_type : kie
[2023/04/14 16:25:17] ppocr INFO: Eval : 
[2023/04/14 16:25:17] ppocr INFO:     dataset : 
[2023/04/14 16:25:17] ppocr INFO:         data_dir : dataset/lxsxkz/lxsxkz/
[2023/04/14 16:25:17] ppocr INFO:         label_file_list : ['dataset/lxsxkz/train.json']
[2023/04/14 16:25:17] ppocr INFO:         name : SimpleDataSet
[2023/04/14 16:25:17] ppocr INFO:         transforms : 
[2023/04/14 16:25:17] ppocr INFO:             DecodeImage : 
[2023/04/14 16:25:17] ppocr INFO:                 channel_first : False
[2023/04/14 16:25:17] ppocr INFO:                 img_mode : RGB
[2023/04/14 16:25:17] ppocr INFO:             VQATokenLabelEncode : 
[2023/04/14 16:25:17] ppocr INFO:                 algorithm : LayoutXLM
[2023/04/14 16:25:17] ppocr INFO:                 class_path : dataset/lxsxkz/class_list.txt
[2023/04/14 16:25:17] ppocr INFO:                 contains_re : True
[2023/04/14 16:25:17] ppocr INFO:                 order_method : tb-yx
[2023/04/14 16:25:17] ppocr INFO:                 use_textline_bbox_info : True
[2023/04/14 16:25:17] ppocr INFO:             VQATokenPad : 
[2023/04/14 16:25:17] ppocr INFO:                 max_seq_len : 512
[2023/04/14 16:25:17] ppocr INFO:                 return_attention_mask : True
[2023/04/14 16:25:17] ppocr INFO:             VQAReTokenRelation : None
[2023/04/14 16:25:17] ppocr INFO:             VQAReTokenChunk : 
[2023/04/14 16:25:17] ppocr INFO:                 max_seq_len : 512
[2023/04/14 16:25:17] ppocr INFO:             TensorizeEntitiesRelations : None
[2023/04/14 16:25:17] ppocr INFO:             Resize : 
[2023/04/14 16:25:17] ppocr INFO:                 size : [224, 224]
[2023/04/14 16:25:17] ppocr INFO:             NormalizeImage : 
[2023/04/14 16:25:17] ppocr INFO:                 mean : [123.675, 116.28, 103.53]
[2023/04/14 16:25:17] ppocr INFO:                 order : hwc
[2023/04/14 16:25:17] ppocr INFO:                 scale : 1
[2023/04/14 16:25:17] ppocr INFO:                 std : [58.395, 57.12, 57.375]
[2023/04/14 16:25:17] ppocr INFO:             ToCHWImage : None
[2023/04/14 16:25:17] ppocr INFO:             KeepKeys : 
[2023/04/14 16:25:17] ppocr INFO:                 keep_keys : ['input_ids', 'bbox', 'attention_mask', 'token_type_ids', 'entities', 'relations']
[2023/04/14 16:25:17] ppocr INFO:     loader : 
[2023/04/14 16:25:17] ppocr INFO:         batch_size_per_card : 8
[2023/04/14 16:25:17] ppocr INFO:         drop_last : False
[2023/04/14 16:25:17] ppocr INFO:         num_workers : 0
[2023/04/14 16:25:17] ppocr INFO:         shuffle : False
[2023/04/14 16:25:17] ppocr INFO: Global : 
[2023/04/14 16:25:17] ppocr INFO:     cal_metric_during_train : False
[2023/04/14 16:25:17] ppocr INFO:     distributed : False
[2023/04/14 16:25:17] ppocr INFO:     epoch_num : 130
[2023/04/14 16:25:17] ppocr INFO:     eval_batch_step : [0, 19]
[2023/04/14 16:25:17] ppocr INFO:     infer_img : ppstructure/docs/kie/input/zh_val_21.jpg
[2023/04/14 16:25:17] ppocr INFO:     kie_det_model_dir : None
[2023/04/14 16:25:17] ppocr INFO:     kie_rec_model_dir : None
[2023/04/14 16:25:17] ppocr INFO:     log_smooth_window : 10
[2023/04/14 16:25:17] ppocr INFO:     print_batch_step : 10
[2023/04/14 16:25:17] ppocr INFO:     save_epoch_step : 2000
[2023/04/14 16:25:17] ppocr INFO:     save_inference_dir : None
[2023/04/14 16:25:17] ppocr INFO:     save_model_dir : ./output/re_vi_layoutxlm_lxsxkz_zh
[2023/04/14 16:25:17] ppocr INFO:     save_res_path : ./output/re/lxsxkz_zh/with_gt
[2023/04/14 16:25:17] ppocr INFO:     seed : 2022
[2023/04/14 16:25:17] ppocr INFO:     use_gpu : True
[2023/04/14 16:25:17] ppocr INFO:     use_visualdl : True
[2023/04/14 16:25:17] ppocr INFO: Loss : 
[2023/04/14 16:25:17] ppocr INFO:     key : loss
[2023/04/14 16:25:17] ppocr INFO:     name : LossFromOutput
[2023/04/14 16:25:17] ppocr INFO:     reduction : mean
[2023/04/14 16:25:17] ppocr INFO: Metric : 
[2023/04/14 16:25:17] ppocr INFO:     main_indicator : hmean
[2023/04/14 16:25:17] ppocr INFO:     name : VQAReTokenMetric
[2023/04/14 16:25:17] ppocr INFO: Optimizer : 
[2023/04/14 16:25:17] ppocr INFO:     beta1 : 0.9
[2023/04/14 16:25:17] ppocr INFO:     beta2 : 0.999
[2023/04/14 16:25:17] ppocr INFO:     clip_norm : 10
[2023/04/14 16:25:17] ppocr INFO:     lr : 
[2023/04/14 16:25:17] ppocr INFO:         learning_rate : 5e-05
[2023/04/14 16:25:17] ppocr INFO:         warmup_epoch : 10
[2023/04/14 16:25:17] ppocr INFO:     name : AdamW
[2023/04/14 16:25:17] ppocr INFO:     regularizer : 
[2023/04/14 16:25:17] ppocr INFO:         factor : 0.0
[2023/04/14 16:25:17] ppocr INFO:         name : L2
[2023/04/14 16:25:17] ppocr INFO: PostProcess : 
[2023/04/14 16:25:17] ppocr INFO:     name : VQAReTokenLayoutLMPostProcess
[2023/04/14 16:25:17] ppocr INFO: Train : 
[2023/04/14 16:25:17] ppocr INFO:     dataset : 
[2023/04/14 16:25:17] ppocr INFO:         data_dir : dataset/lxsxkz/lxsxkz/
[2023/04/14 16:25:17] ppocr INFO:         label_file_list : ['dataset/lxsxkz/train.json']
[2023/04/14 16:25:17] ppocr INFO:         name : SimpleDataSet
[2023/04/14 16:25:17] ppocr INFO:         ratio_list : [1.0]
[2023/04/14 16:25:17] ppocr INFO:         transforms : 
[2023/04/14 16:25:17] ppocr INFO:             DecodeImage : 
[2023/04/14 16:25:17] ppocr INFO:                 channel_first : False
[2023/04/14 16:25:17] ppocr INFO:                 img_mode : RGB
[2023/04/14 16:25:17] ppocr INFO:             VQATokenLabelEncode : 
[2023/04/14 16:25:17] ppocr INFO:                 algorithm : LayoutXLM
[2023/04/14 16:25:17] ppocr INFO:                 class_path : dataset/lxsxkz/class_list.txt
[2023/04/14 16:25:17] ppocr INFO:                 contains_re : True
[2023/04/14 16:25:17] ppocr INFO:                 order_method : tb-yx
[2023/04/14 16:25:17] ppocr INFO:                 use_textline_bbox_info : True
[2023/04/14 16:25:17] ppocr INFO:             VQATokenPad : 
[2023/04/14 16:25:17] ppocr INFO:                 max_seq_len : 512
[2023/04/14 16:25:17] ppocr INFO:                 return_attention_mask : True
[2023/04/14 16:25:17] ppocr INFO:             VQAReTokenRelation : None
[2023/04/14 16:25:17] ppocr INFO:             VQAReTokenChunk : 
[2023/04/14 16:25:17] ppocr INFO:                 max_seq_len : 512
[2023/04/14 16:25:17] ppocr INFO:             TensorizeEntitiesRelations : None
[2023/04/14 16:25:17] ppocr INFO:             Resize : 
[2023/04/14 16:25:17] ppocr INFO:                 size : [224, 224]
[2023/04/14 16:25:17] ppocr INFO:             NormalizeImage : 
[2023/04/14 16:25:17] ppocr INFO:                 mean : [123.675, 116.28, 103.53]
[2023/04/14 16:25:17] ppocr INFO:                 order : hwc
[2023/04/14 16:25:17] ppocr INFO:                 scale : 1
[2023/04/14 16:25:17] ppocr INFO:                 std : [58.395, 57.12, 57.375]
[2023/04/14 16:25:17] ppocr INFO:             ToCHWImage : None
[2023/04/14 16:25:17] ppocr INFO:             KeepKeys : 
[2023/04/14 16:25:17] ppocr INFO:                 keep_keys : ['input_ids', 'bbox', 'attention_mask', 'token_type_ids', 'entities', 'relations']
[2023/04/14 16:25:17] ppocr INFO:     loader : 
[2023/04/14 16:25:17] ppocr INFO:         batch_size_per_card : 16
[2023/04/14 16:25:17] ppocr INFO:         drop_last : False
[2023/04/14 16:25:17] ppocr INFO:         num_workers : 0
[2023/04/14 16:25:17] ppocr INFO:         shuffle : True
[2023/04/14 16:25:17] ppocr INFO: profiler_options : None
[2023/04/14 16:25:17] ppocr INFO: train with paddle 2.4.0 and device Place(gpu:0)
[2023/04/14 16:25:17] ppocr INFO: Initialize indexs of datasets:['dataset/lxsxkz/train.json']
[2023-04-14 16:25:18,760] [    INFO] - Already cached /home/aistudio/.paddlenlp/models/layoutxlm-base-uncased/sentencepiece.bpe.model
[2023-04-14 16:25:19,506] [    INFO] - tokenizer config file saved in /home/aistudio/.paddlenlp/models/layoutxlm-base-uncased/tokenizer_config.json
[2023-04-14 16:25:19,506] [    INFO] - Special tokens file saved in /home/aistudio/.paddlenlp/models/layoutxlm-base-uncased/special_tokens_map.json
[2023/04/14 16:25:19] ppocr INFO: Initialize indexs of datasets:['dataset/lxsxkz/train.json']
[2023-04-14 16:25:19,510] [    INFO] - Already cached /home/aistudio/.paddlenlp/models/layoutxlm-base-uncased/sentencepiece.bpe.model
[2023-04-14 16:25:20,257] [    INFO] - tokenizer config file saved in /home/aistudio/.paddlenlp/models/layoutxlm-base-uncased/tokenizer_config.json
[2023-04-14 16:25:20,257] [    INFO] - Special tokens file saved in /home/aistudio/.paddlenlp/models/layoutxlm-base-uncased/special_tokens_map.json
[2023-04-14 16:25:20,261] [    INFO] - Already cached /home/aistudio/.paddlenlp/models/vi-layoutxlm-base-uncased/model_state.pdparams
W0414 16:25:20.264479 15787 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.0, Driver API Version: 11.2, Runtime API Version: 11.2
W0414 16:25:20.268268 15787 gpu_resources.cc:91] device: 0, cuDNN Version: 8.2.
[2023/04/14 16:25:24] ppocr INFO: train dataloader has 4 iters
[2023/04/14 16:25:24] ppocr INFO: valid dataloader has 8 iters
[2023/04/14 16:25:24] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 19 iterations

**Comment by smallhaozigithub** [comment]
根据train.py文件一路找代码，发现是运行到tools/program.py  288行
![image](https://user-images.githubusercontent.com/72907766/231990183-1ef74bc8-22a7-4fe3-a381-faf833442ca8.png)
模型没有返回结果。

**Comment by happybuby** [comment]
是不是把原来的question和answer两个类别给替换成其它的了，替换后好像就会出现这种情况

**Comment by smallhaozigithub** [comment]
label用的是question 和answer 。

**Comment by smallhaozigithub** [comment]
还木有，我是ser可以训练，训练re的时候出现这个问题。

**Comment by lbeing** [comment]
可能数据集有问题，特别新加的linking部分，因为ser不需要这个
